src/TicketingSystem.DataAccess/Entities/Event.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+using TicketingSystem.DataAccess.Enums;
+
+namespace TicketingSystem.DataAccess.Entities
+{
+    public class Event : BaseEntity
+    {
+        public string Name { get; set; }
+
+        public string Description { get; set; }
+
+        public DateTimeOffset StartTime { get; set; }
+
+        public DateTimeOffset EndTime { get; set; }
```

Comment:
These DateTime properties look in my local DB like this:
![2024-04-28_15h12_43](https://github.com/leg0ushque/ticket-management/assets/58484111/c330cbb3-f5d8-4ab5-b740-8fa1960d70d5)
Is it expected? Will it be convenient to analyze DB queries or just data looking into the document in the future? Maybe you can use `[BsonRepresentation(BsonType.DateTime)]` attribute on these properties and then have in DB something like this:
![2024-04-28_15h15_56](https://github.com/leg0ushque/ticket-management/assets/58484111/5a020f3b-ae02-45f5-a938-cd2747d046e9)

Reply:
Fixed